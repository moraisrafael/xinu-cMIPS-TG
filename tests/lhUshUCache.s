	.include "cMIPS.s"
	.text
	.align 2
	.globl _start
	.ent _start
_start: la    $15, x_DATA_BASE_ADDR
	la    $26, x_IO_BASE_ADDR
	addi  $3,$0, -16
	nop
snd:	sh   $3, 0($15)
	addi $3,$3,1
	lhu  $4, 0($15)
	addi $15,$15,2
	sw   $4, 0($26)
	bne  $3,$0,snd
	nop
	nop
	nop
	nop
	wait
	nop
	.end _start

	#lbu   $4, 0($15)



	