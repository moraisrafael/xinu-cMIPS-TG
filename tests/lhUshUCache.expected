0000fff0
0000fff1
0000fff2
0000fff3
0000fff4
0000fff5
0000fff6
0000fff7
0000fff8
0000fff9
0000fffa
0000fffb
0000fffc
0000fffd
0000fffe
0000ffff
